src/Application/Storage/Queries/StorageList.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Hippo.Application.Revisions.Queries;
+
+public class StorageList
+{
+    public StorageList(List<string?> storages)
+    {
+        Storages = storages;
+    }
+
+    [Required]
+    public List<string?> Storages { get; set; }
```

Comment:
What value would a null string provide in this case? Is there anything we can reasonably do with that data?